swagger.yaml

```diff
@@ -4171,27 +8119,27 @@ definitions:
         format: date-time
         x-go-custom-tag: gorm:"type:timestamp with time zone"
         description: The last time that the cluster status was updated.
-      hosts:
+      agents:
```

Comment:
lets rename it to cluster_agents and set the tag to: 'gorm:"-"'
we are goin g to add Agents to the common.Cluster, so that we will load the infra_env with preload, but the field itself won't be in the models

Reply:
Why do we need to rename to cluster_agents?